app/src/Chat.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from "react";
+import "./App.css";
+
+type ChatProps = {
+  socket: any;
```

Comment:
it is not advised to use `any`, refrain from it unless you don't have any clue on what the type might be.

Type this to `Socket` type from the `socket.io` library